branca/element.py

```diff
@@ -320,43 +320,26 @@ def render(self, **kwargs):
         return self._template.render(this=self, kwargs=kwargs)
 
     def _repr_html_(self, **kwargs):
-        """Displays the Figure in a Jupyter notebook.
-
-        Percent-encoded HTML is stored in data-html attribute, which is used to populate
-        the iframe. This approach does not encounter the 2MB limit in Chrome for storing
-        the HTML in the src attribute with a data URI. The alternative of using a srcdoc
-        attribute is not supported in Microsoft Internet Explorer and Edge.
-
-        """
-        html = urllib.parse.quote(self.render(**kwargs))
-        onload = (
-            'this.contentDocument.open();'
-            'this.contentDocument.write('
-            '    decodeURIComponent(this.getAttribute(\'data-html\'))'
-            ');'
-            'this.contentDocument.close();'
-        )
-
+        """Displays the Figure in a Jupyter notebook."""
+        html = escape(self.render(**kwargs))
         if self.height is None:
             iframe = (
                 '<div style="width:{width};">'
                 '<div style="position:relative;width:100%;height:0;padding-bottom:{ratio};">'  # noqa
                 '<span style="color:#565656">Make this Notebook Trusted to load map: File -> Trust Notebook</span>'  # noqa
```

Comment:
I see that this line was added in PR #75 to resolve an issue from the Javascript handling added in PR #66.

With a `srcdoc`, can this line now be deleted?

```suggestion
```

If this line is still necessary (or retained as a precaution), should it also be added below for the `iframe` in the `else` clause.

Reply:
Notebooks still have to be trusted to be able to fill the iframe, even when using `srcdoc`. So let's keep the message.

We don't need it in the `else` clause, because that state is only reached for non-default settings, in which case I trust we don't have to warn users for this.